src/EventPayload.ts

```diff
@@ -111,6 +111,8 @@ export interface EventPayload {
   sessionId?: string | null;
   /** Fields specific to reporting Yext Pages Analytics Events */
   pages?: {
+    /** The scope of an action. Applies to Pages events. */
```

Comment:
Are we holding off adding tests using this until we implement the backend?

Reply:
That's what I was thinking, but do you think we can go ahead and add tests?